config/prow/cluster/monitoring/grafana_deployment.yaml

```diff
@@ -30,6 +34,8 @@ spec:
         ports:
           - containerPort: 80
```

Comment:
We might have to run with `CAP_NET_BIND_SERVICE` to bind low ports when running as non-root, depending on whether the Docker version is 1.19 or 1.20 (docker set `net.ipv4.ip_unprivileged_port_start=0` in 1.20, and I'm not sure what containerd does): https://github.com/moby/moby/pull/41030

Reply:
I added a `capabilities` section to the pod's `securityContext` that I think should grant that capability.

Reply:
You could also just bind something higher and use the service to redirect it